src/app.js

```diff
@@ -1 +1,29 @@
 'use strict';
+
+const fs = require('fs/promises');
+const colors = require('colors');
+const moment = require('moment');
+
+colors.enable();
+
+// eslint-disable-next-line space-before-function-paren
+const createFile = async (fileName, content) => {
+  try {
+    const file = await fs.appendFile(fileName, content);
+
+    // eslint-disable-next-line no-console
+    console.log(fileName.replace(/.\/src\//, '') + ' - ' + content);
```

Comment:
always prefer string literal
```suggestion
    console.log(`${fileName.replace(/.\/src\//, '')} - ${content}`);
```